### PR TITLE
- fix #913

### PR DESCRIPTION
**Pull request by ivanvolkoff**
- remove type artifact from calculationview/hdbcalculationview

### CODE REVIEW

File: ide-migration/server/migration/api/migration-service.js [hunk 1]
```diff
@@ -270,7 +272,15 @@ class MigrationService {
         const project = workspace.getProject(projectName)
         const projectFile = project.createFile(relativePath);
         const resource = repositoryManager.getResource(repositoryPath);
-        projectFile.setContent(resource.getContent());
+        if (relativePath.endsWith('hdbcalculationview') || relativePath.endsWith('calculationview') || repositoryPath.endsWith('hdbcalculationview') || repositoryPath.endsWith('calculationview')) {
+            try {
+                projectFile.setContent(calculationViewModifier.removeTypeArtifact(resource.getContent()));
+            } catch (e) {
+                throw new Error(e)
```
Comment (vmutafov) [inline ide-migration/server/migration/api/migration-service.js:279]: Do we need this rethrow?

File: ide-migration/server/migration/api/migration-service.js [hunk 2]
```diff
@@ -270,7 +271,12 @@ class MigrationService {
         const project = workspace.getProject(projectName)
         const projectFile = project.createFile(relativePath);
         const resource = repositoryManager.getResource(repositoryPath);
-        projectFile.setContent(resource.getContent());
+
+        if (relativePath.endsWith('hdbcalculationview') || relativePath.endsWith('calculationview') || repositoryPath.endsWith('hdbcalculationview') || repositoryPath.endsWith('calculationview')) {
+            projectFile.setContent(calculationViewModifier.removeTypeArtifact(resource.getContent()));
+        } else {
+            projectFile.setContent(resource.getContent());
+        }
```
Comment (ThuF) [inline ide-migration/server/migration/api/migration-service.js:279]: FYI: @vmutafov, @ivanvolkoff

Ideally this logic block should be extracted into separate process tasks, as it's expected to have also several others transformations happening as part of the file content migration/modification.

**Review by ThuF — APPROVED** [review]
+1 seems good to me

File: ide-migration/server/migration/api/migration-service.js (hunk 2 above)
Comment (ivanvolkoff) [reply ide-migration/server/migration/api/migration-service.js:279]: We are currently working on extracting the logic for transformation/generation files to a different place.

Comment (ThuF) [reply ide-migration/server/migration/api/migration-service.js:279]: @ivanvolkoff Great! Should we merge the PR, or there are upcoming changes that we should wait for?

**Review by vmutafov — APPROVED** [review]

File: ide-migration/server/migration/api/migration-service.js (hunk 2 above)
Comment (vmutafov) [reply ide-migration/server/migration/api/migration-service.js:279]: I merged this one, I’ll create a PR for the XSK right after lunch

Comment (ThuF) [reply ide-migration/server/migration/api/migration-service.js:279]: Thanks @ivanvolkoff and @vmutafov!

**Pull request merged**